src/domain/delegate/delegate.repository.ts

```diff
@@ -36,4 +37,24 @@ export class DelegateRepository implements IDelegateRepository {
     page?.results.map((result) => this.delegateValidator.validate(result));
     return page;
   }
+
+  async postDelegates(
+    chainId: string,
+    safeAddress?: string,
+    delegate?: string,
+    delegator?: string,
+    signature?: string,
+    label?: string,
+  ): Promise<NetworkResponse<any>> {
```

Comment:
The domain should not expose data-source related entities.

If we don't care about the object returned we should probably use `unknown` here – `Promise<unknown>`